lms/djangoapps/certificates/api.py

```diff
@@ -172,6 +173,14 @@ def generate_user_certificates(student, course_key, course=None, insecure=False,
         forced_grade - a string indicating to replace grade parameter. if present grading
                        will be skipped.
     """
+    users_queryset = list_with_level(course, u'beta')
+    username_list = [user.username for user in users_queryset]
+
+    if student.username in username_list:
+        message = u'Beta Tester is not allowed to generate course certificate: course_id: {}, username: {}'
```

Comment:
nit: `u'Cancelling course certificate generation for user [{}] against course [{}], user is a Beta Tester.'`